packages/nodejs/bin/diagnose

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+const https = require("https")
+const { DiagnoseTool } = require("../dist/diagnose")
+
+// enable diagnose mode
+process.env["_APPSIGNAL_DIAGNOSE"] = "true"
+
+const tool = new DiagnoseTool({})
+
+console.log(`
+🔧 AppSignal Diagnose Tool
+
+Use this information to debug your configuration.
+More information is available on the documentation site.
+https://docs.appsignal.com/
+
+Send this output to support@appsignal.com if you need help.
```

Comment:
Since we post the result I suggest this text:
```suggestion
This diagnose output was sent to AppSignal, contact us at support@appsignal.com if you need help.
```